scripts/importmetadata

```diff
@@ -0,0 +1,229 @@
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{project}}"]' >> importmetadata.txt
+					;;
+				google_compute_project_default_network_tier)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{project}}"]' >> importmetadata.txt
+					;;
+				google_compute_project_metadata)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{project}}"]' >> importmetadata.txt
+					;;
+				google_compute_project_metadata_item)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{key}}"]' >> importmetadata.txt
+					;;
+				google_compute_shared_vpc_host_project)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{project}}"]' >> importmetadata.txt
+					;;
+				google_compute_shared_vpc_service_project)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{host_project}}/{{service_project}}"]' >> importmetadata.txt
+					;;
+				google_dataflow_job)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{job_id}}"]' >> importmetadata.txt
+					;;
+				google_billing_subaccount)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["billingAccounts/{{master_billing_account}}"]' >> importmetadata.txt
+					;;
+				google_organization_iam_custom_role)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["organizations/{{org_id}}/roles/{{role_id}}"]' >> importmetadata.txt
+					;;
+				google_storage_notification)
+					echo -n \	\"$resource\"\: >> importmetadata.txt
+					echo \ '["{{notification_id}}", "{{bucket}}/notificationConfigs/{{id}}"]' >> importmetadata.txt
+					;;
+				google_dataflow_flex_template_job)
+#code checks for importstatepassthrough but officially does not support importing so we proceed with no entry
+					;;
+				google_compute_disk_async_replication)
+#code checks for importstatepassthrough but officially does not support importing so we proceed with no entry
+					;;
+				*)
+#resource needs to be hardcoded into this script for support
+					echo -n "$resource: " >> importmetadata.txt
+					echo \ NOT CONFIGURED  >> importmetadata.txt
+#					exit 1
+				esac
+			fi
+	    else
+			echo -n \	\"$resource\"\: >> importmetadata.txt
+			importformat=${importformat/tpgresource.ParseImportId\(\[\]string\{/\[}
+			importformat=${importformat/\}/]}
+			importformat=${importformat//(?P</\{\{}}
+			importformat=${importformat//\>\[\^\/\]\+\)/\}\}}
+			importformat=${importformat//\>\[\^\/\]\+\)/\}}
+#covering multiple formatting edge cases
+			importformat=${importformat//\>\[\^ \]\+\)/\}\}}
+			importformat=${importformat//\>\.\+\)/\}\}}
+			importformat=${importformat/\]\}/\]}
+			importformat=${importformat//\>\[\^\/\]\*\)/\}\}} [^/]*
+
+			importformat=$(sed -e 's|, \]|\]|g'  <<< $importformat)
+			importformat=$(sed -e 's|\[ \"|\[\"|g'  <<< $importformat)
+
+	    	echo \ "$importformat" >> importmetadata.txt
+	    fi
+	    iamname=${resource/google/iam}
```

Comment:
IAM resources currently look like

```
"iam_cloudfunctions_function": ["projects/{{project}}/locations/{{region}}/functions/{{name}}", "{{project}}/{{region}}/{{name}}", "{{name}}"]
```

Could we make them map to the literal resources? That'll save work for anyone using this, since they'd need to translate themselves right now:

```
"google_cloudfunctions_function_iam_policy": ["projects/{{project}}/locations/{{region}}/functions/{{name}}", "{{project}}/{{region}}/{{name}}", "{{name}}"]
"google_cloudfunctions_function_iam_binding": ["projects/{{project}}/locations/{{region}}/functions/{{name}} {{role}}", "{{project}}/{{region}}/{{name}} {{role}}", "{{name}} {{role}}"]
"google_cloudfunctions_function_iam_member": ["projects/{{project}}/locations/{{region}}/functions/{{name}} {{role}} {{member}}", "{{project}}/{{region}}/{{name}} {{role}} {{member}}", "{{name}} {{role}} {{member}}"]
```

(really, we should insert those into our import blocks in the IAM resources too, rather than fake values. we used to use fake values more often.)